### PR TITLE
fix!: deprecate YAML configuration

### PR DESCRIPTION
**Pull request by danielbrunt57**
YAML configuration is deprecated. Users must manually remove `alexa_media` from `configuration.yaml` file and add the integration via UI to avoid warning messages and future failure.

BREAKING CHANGE: Deprecate YAML configuration

### CODE REVIEW

**Comment by danielbrunt57** [comment]
Requires PR #2681

**Comment by alandtse** [comment]
To be clear, the only changes to strings should be related to the deprecation.

File: custom_components/alexa_media/strings.json [hunk 1]
```diff
@@ -102,5 +88,22 @@
...
+          "confirm_restart": {
+            "title": "Restart required",
+            "description": "Restart of Home Assistant is required to finish download/update of {name}, click submit to restart now."
+          }
+        }
+      }
+    },
+    "deprecated_yaml_configuration": {
+      "title": "YAML configuration is deprecated",
+      "description": "YAML configuration of Alexa Media Player is deprecated\nand will be removed in version 4.14.0.\nThere will be no automatic import of this.\nPlease remove it from your configuration, restart Home Assistant and use the UI to configure it instead.\nSettings > Devices & services > Integrations > ADD INTEGRATION"
```
Comment (alandtse) [inline custom_components/alexa_media/strings.json:109]: Everything else in the string.json is inappropriate for this commit.

File: custom_components/alexa_media/strings.json [hunk 2]
```diff
@@ -102,5 +102,22 @@
         }
       }
     }
+  },
+  "issues": {
+    "restart_required": {
```
Comment (alandtse) [inline custom_components/alexa_media/strings.json]: Where is this used?

Comment (danielbrunt57) [reply custom_components/alexa_media/strings.json]: I copied code from HACS when they deprecated YAML but I realize now that my other edits don't use it (now that you've pointed it out) and have removed that section.

**Review by alandtse — APPROVED** [review]

**Pull request merged**